documentation/docs/get-started/demo-explorer-get-started/basic-operations.md

```diff
@@ -2,30 +2,41 @@
 sidebar_label: 'Basic Operations'
 sidebar_position: 3
 ---
+
 # Basic Operations
+
 :::info
-To try this locally, you need to create a secure tunnel to your `localhost:3332`. You can use **ngrok** to do this.
+To try this locally, you need to create a secure tunnel to your `localhost:3332`. You can use [**ngrok**](/docs/get-started/api-server-get-started/basic-operations#optional-using-https-for-localhost-by-using-ngrok) to do this.
```

Comment:
Hi @namhoang1604, as I understand, when clicking on `ngork` it should redirect `https://ngrok.com/docs/getting-started/` in another tab. (#491)

Reply:
I enhanced it to add more information about why we need to use ngrok